events.js

```diff
@@ -66,12 +66,22 @@ const events2019 = [
   }
 ];
 
-function initEventComponents() {
-  Vue.component('event-box', {
-    props: ['event'],
+const yearlyEvents = [
+  {
+    year: 2021,
+    events: events2021 
+  }, {
+    year: 2019,
+    events: events2019
+  }
+];
+
+function initGalleryComponents() {
+  Vue.component('gallery', {
+    props: ['event', 'showTitle'],
     template: '<div class="imageitem">' +
               '  <v-card elevation="2" class="gallery">' +
-              '    <v-card-title>{{ `${event.name}, ${event.date}` }}</v-card-title>' +
+              '    <v-card-title v-if=showTitle>{{ `${event.name}, ${event.date}` }}</v-card-title>' +

```

Comment:
When it's used by concerts, we don't need a title above the gallery, since a concert has its title at the top of concert-box.